volatility/framework/plugins/windows/procdump.py

```diff
@@ -35,60 +35,53 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
         ]
 
     @classmethod
-    def process_dump(cls, proc)
-        pe_table_name = intermed.IntermediateSymbolTable.create(self.context,
-                                                                self.config_path,
+    def process_dump(cls, proc, context: interfaces.context.ContextInterface, config_path: str, config: dict):
```

Comment:
Sorry I didn't mention this earlier, but we try to request the context first, then things like `symbol_table_name` and then finally the `proc`.  As for the `config_path`, it's only used for creating the PE table (which only needs to happen once), so I'd suggest pulling that out of this method, and adding a parameter `pe_table_name`.

So I'd expect the final method signature to be something like:
```
def process_dump(cls, context: interfaces.context.ContextInterface, kernel_table_name: str, pe_table_name: str, proc: interfaces.objects.ObjectInterface) -> interfaces.plugins.FileInterface:
   """Extracts the complete data for a process as a FileInterface

   Args:
     context: the context to operate upon
     kernel_table_name: the name for the symbol table containing the kernel's symbols
     pe_table_name: the name for the symbol table containing the PE format symbols
     proc: the process object whose memory should be output

   Returns:
     A FileInterface object containing the complete data for the process
"""
```

Sorry to keep giving more specific requests, but we still haven't gotten a style guide in place, so sadly it's just in my head so I end up forgetting to mention bits that we'd prefer to be done a particular way.  Sorry!  5:S